lighthouse-core/gather/gatherers/form-elements.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * @license Copyright 2019 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Gatherer = require('./gatherer.js');
+const pageFunctions = require('../../lib/page-functions.js');
+
+/* eslint-env browser, node */
+
+
+/**
+ *  @param {HTMLFormElement}
+ *  @return {any[]}
```

Comment:
Instead of the return type being 'any[]' for this and the function below it, you should explicitly define the type. In this case, I believe `@return {Array<HTMLElement>}` could work. It'd also remove the need for you to cast later on in the map functions.